texts/en_US.properties

```diff
@@ -18,6 +18,9 @@ geyser.auth.login.offline=Attempting to login using offline mode... authenticati
 geyser.auth.login.wait=Please wait until you are logged in...
 geyser.auth.stored_credentials=Using stored credentials for bedrock user {0}
 
+geyser.bootstrap.args.confignotspecified=Please specify config file with -c option.
+geyser.bootstrap.args.unrecognised=Unrecognised argument \'{0}\'
+geyser.bootstrap.args.usage=Usage: [java -jar] Geyser.jar [opts]\n  Options:\n    -c, --config [file]    use file for config\n    -h, --help             show this help message\n    --gui, --nogui         enable\/disable gui
```

Comment:
maybe move this to separate strings `Usage: {0}` and then one for each description EG `use file for config` so when more are added we don't have to wait for all languages to update to display them

Reply:
Sounds good.